tensorflow/compiler/xla/service/hlo_instruction.cc

```diff
@@ -1788,9 +1788,9 @@ HloInstruction::CreateDynamicReshape(
 /* static */ std::unique_ptr<HloInstruction> HloInstruction::CreateFusion(
     const Shape& shape, FusionKind fusion_kind,
     absl::Span<HloInstruction* const> operands,
-    HloComputation* fusion_computation) {
+    HloComputation* fusion_computation, std::string prefix) {
```

Comment:
The "prefix" logic probably deserves it's own separate CL

Reply:
If I do this, the PR will add a new feature that isn't used or tested. I was told that this isn't good normally.

I can still split it if you want, but too many small PR that depends on each other is also trouble some to handle and cause merge delays.

Reply:
Yes, ideally it should be a PR depending on this one, but github does not have tooling for such workflow. Could we still split to separate naming changes from optimization? The other PR should come after this one.